dell/inspiron/3442/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, ... }:
+	
+{
+	imports	= [
+		../../../common/cpu/intel
+		../../../common/pc/laptop
+	];
+
+  hardware = {
+    graphics = {
+			enable = lib.mkDefault true;
+      extraPackages = with pkgs; [
+        intel-media-driver # LIBVA_DRIVER_NAME=iHD
+        vaapiVdpau
+        libvdpau-va-gl
+      ];
+    };
+		enableRedistributableFirmware = lib.mkDefault true;
+	};
+
+	services = {
+		fwupd =	{
+			enable = lib.mkDefault true;
+		};
+		thermald = {
+			enable = lib.mkDefault true;
+		};
+    tlp = {
```

Comment:
We already enable this in the laptop profile.

Reply:
removed